facade/nodeFacade.go

```diff
@@ -220,8 +216,10 @@ func (nf *nodeFacade) startRest() {
 			"SameSourceResetIntervalInSec", nf.wsAntifloodConfig.SameSourceResetIntervalInSec,
 		)
 
-		responseLoggerMiddleware := middleware.NewResponseLoggerMiddleware(thresholdDurationForLoggingRequests)
-		limiters = append(limiters, responseLoggerMiddleware)
+		if nf.apiRoutesConfig.Logging.LoggingEnabled {
+			responseLoggerMiddleware := middleware.NewResponseLoggerMiddleware(time.Duration(nf.apiRoutesConfig.Logging.ThresholdInNanoSeconds) * time.Nanosecond)
```

Comment:
:+1: I think this is the most efficient way to enable/disable the logging.

Reply:
Yes. Also, if one forgets to update the `api.toml` then the default value of `false` will apply so no logging